hotdoc/extensions/gi/gtkdoc_links.py

```diff
@@ -12,7 +12,7 @@ def parse_devhelp_index(dir_):
     if not os.path.exists(path):
         return False
 
-    dh_root = etree.parse(path).getroot()
+    dh_root = etree.parse(path, parser=etree.XMLParser(huge_tree=True)).getroot()
```

Comment:
The doc for this says "disables security restrictions", afaiu you don't actually need this as you ended up with a huge tree because of a user error? If so, we probably want to keep security restrictions :)

Reply:
Removed commit, whatever indeed.